docker-compose.yml

```
@@ -22,20 +22,13 @@ services:
     - "5001:8080"
     environment:
       <<: *x-env-generic-backend
-      DBHOST: db
-      DBUSER: postgres
-      DBPASS: OL2AEn6lgj6ekajgKJIOanefgegnksngpoetPIEQjhankf7412
-      DBPORT: 5432
-      DBNAME: wharf
-      DBLOG: "true"
-      RABBITMQUSER: guest
-      RABBITMQPASS: guest
-      RABBITMQHOST: rabbitmq
-      RABBITMQPORT: 5672
-      RABBITMQVHOST:
-      RABBITMQNAME: wharf_queue
-      RABBITMQDISABLESSL: "true"
-      RABBITMQCONNATTEMPTS: 10
+      WHARF_HTTP_DB_DRIVER: postgres
+      WHARF_HTTP_DB_HOST: db
+      WHARF_HTTP_DB_PORT: 5432
+      WHARF_HTTP_DB_USERNAME: postgres
+      WHARF_HTTP_DB_PASSWORD: OL2AEn6lgj6ekajgKJIOanefgegnksngpoetPIEQjhankf7412
+      WHARF_HTTP_DB_NAME: wharf
+      WHARF_HTTP_DB_LOG: "true"
```

In trying it out I realized these are all wrong. Shouldn't have `_HTTP`
```suggestion
      WHARF_DB_DRIVER: postgres
      WHARF_DB_HOST: db
      WHARF_DB_PORT: 5432
      WHARF_DB_USERNAME: postgres
      WHARF_DB_PASSWORD: OL2AEn6lgj6ekajgKJIOanefgegnksngpoetPIEQjhankf7412
      WHARF_DB_NAME: wharf
      WHARF_DB_LOG: "true"
```